src/com/rakovets/course/practice/module4/Task01.java

```diff
@@ -22,5 +22,12 @@ public static void main(String[] args) {
 		//TODO
 		// Код, решающий задачу пишем ниже, при этом используяся переменные объявленные выше (их можно изменять)
 		// Для проверки решения необходимо запустить @Test для данного class (в директории test)
+		for (int i = 1; i <= rows; i++) {
+			if (rows != i) {
+				System.out.print(i + "\n");
+			} else {
+				System.out.print(i);
+			}

```

Comment:
Best practice using ternary operator
```suggestion
			System.out.print((rows != i) ? i + "\n" : i);
```